packages/components/textarea/src/textarea.stories.ts

```diff
@@ -32,20 +29,19 @@ export const API: StoryObj = {
     },
     resize: {
       control: 'inline-radio',
-      options: resizeTypes
+      options: ['none', 'vertical', 'auto']
     }
   },
-  render: ({ disabled, placeholder, required, size, value, resize, readonly, hint }) =>
+  render: ({ disabled, placeholder, required, size, value, resize, readonly }) =>
     html`
       <sl-textarea
         ?readonly=${readonly}
         .disabled=${disabled}
         .placeholder=${placeholder}
         .required=${required}
+        .resize=${resize}
         .size=${size}
         .value=${value}
-        .resize=${resize}
-        .hint=${hint}
       ></sl-textarea>
     `
 };
```

Comment:
Can you please also update `All` story?
<img width="933" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/9e209acb-8401-4665-9471-e7c28e95cfb8">
There should be an icon when the textarea is invalid and also there are some examples that should have visually shown that the textarea is valid (with green icon)